code/game/objects/items/toys.dm

```diff
@@ -438,16 +437,6 @@
 	item_state = "egg3" // It's the green egg in items_left/righthand
 	item_color = "green"
 
-/obj/item/toddler
-	icon_state = "toddler"
```

Comment:
Sprite is being utilized by anything else? If not then remove it.

Reply:
Sure, will also be done by the end of the night.

Reply:
icon removed